src/api/config/routes.rb

```diff
@@ -572,9 +572,12 @@ def self.public_or_about_path?(request)
     resources :architectures, only: [:index, :show, :update] # create,delete currently disabled
 
     ### /trigger
-    post 'trigger/runservice' => 'trigger#runservice'
+    post 'trigger/runservice' => 'services/trigger#runservice'
     post 'trigger/webhook' => 'services/webhooks#create'
 
+    ### /services
+    put 'services/add_service_file' => 'services/base#add_service_file'
```

Comment:
We use RESTful routes:-) That means that the action, like 'add', is defined via the HTTP Verb, here PUT. So it's not necessary to have it as part of the URL.
Usually we would write this like `put 'services/service_file' => 'services/service_file#create'` or even drop the `_service` part -> `services/file`. That would mean that there would be a Service::ServiceFileController Service::FileController with a create action.

Some of our [webui controllers](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/controllers/webui/projects/maintenance_incidents_controller.rb) be a good example for this.

Reply:
@bgeuken makes sense, I will change it accordingly :)